apps/web-client/src/lib/db/cr-sqlite/__tests__/supplier-order.test.ts

```diff
@@ -301,6 +330,32 @@ describe("New supplier orders:", () => {
 			expect(orderLine.authors).toBe("N/A");
 			expect(orderLine.line_price).toBe(0);
 		});
+
+		// In case of overdelivering, a reconcfiliation order might spill over and assign
+		// overdelivered books to customer order lines that haven't been placed with a supplier.
+		// Those books won't be marked as placed, but will be marked as 'received' - this test is here to ensure those lines
+		// aren't included in possible order lines
+		it("not include received books", async () => {
```

Comment:
A duplicate?

Reply:
Ah, no...almost identical, but one deals in supplier order, another is supplier order lines